.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor
 /composer.lock
+.idea
```

Comment:
This is probably better placed in your user global gitignore:
https://robots.thoughtbot.com/global-gitignore